File: python/Ejercicio48.py
import math

r11 = float(input("Ingrese el elemento (fila 1, columna 1) de la matriz: "))
r12 = float(input("Ingrese el elemento (fila 1, columna 2) de la matriz: "))
r21 = float(input("Ingrese el elemento (fila 2, columna 1) de la matriz: "))
r22 = float(input("Ingrese el elemento (fila 2, columna 2) de la matriz: "))

angulo_radianes = math.atan2(r21, r11)
angulo_grados = math.degrees(angulo_radianes)

print("El ángulo de rotación es:", angulo_grados, "grados")

File: java/Ejercicio48.java
import java.util.Scanner;


public class Ejercicio48 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
       

        System.out.print("Ingrese el elemento (fila 1, columna 1) de la matriz: ");
        double r11 = sc.nextDouble();

        System.out.print("Ingrese el elemento (fila 1, columna 2) de la matriz: ");
        double r12 = sc.nextDouble();

        System.out.print("Ingrese el elemento (fila 2, columna 1) de la matriz: ");
        double r21 = sc.nextDouble();

        System.out.print("Ingrese el elemento (fila 2, columna 2) de la matriz: ");
        double r22 = sc.nextDouble();

        double anguloRadianes = Math.atan2(r21, r11);
        double anguloGrados = Math.toDegrees(anguloRadianes);

        System.out.println("El ángulo de rotación es: " + anguloGrados + " grados");

        sc.close();
    }
}